src/core/Microsoft.PowerToys.Settings.UI.Lib/PowerLauncherProperties.cs

```diff
@@ -55,6 +55,7 @@ public PowerLauncherProperties()
             IgnoreHotkeysInFullscreen = false;
             DisableDriveDetectionWarning = false;
             ClearInputOnLaunch = false;
+            MaximumNumberOfResults = 9;
```

Comment:
The default value in the code is set to 4. I think we should change it to that.

Reply:
fixed